### ISSUE TITLE
Poor exception message when canvas size is too large

### ISSUE THREAD

**Issue by sandersaares** [Microsoft/Win2D#122]
The following XAML snippet in a blank UWP app template will crash the app, presumably because some limitation on the size of the CanvasControl.

```
	<ScrollViewer Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
		<Grid MinWidth="64000" MinHeight="64000">
			<Xaml:CanvasControl />
		</Grid>
	</ScrollViewer>
```

The issue is that the only feedback the app gives the developer is an exception caught in the global app-level exception handler with the message "The parameter is incorrect.", including no stack trace or anything useful to aid in finding out the real problem.

I would expect there to be at least be some actual error message that points to the problem.

**Comment by shawnhar** [comment]
Thanks for the suggestion - I've added this to our backlog and it will be prioritized during our next sprint planning (on Monday).

**Comment by SebastianStehle** [comment]
Is there an option to get the maximum canvas size and how are the performance characteristics of different sizes?

**Comment by shawnhar** [comment]
Max size for your GPU is available via http://microsoft.github.io/Win2D/html/P_Microsoft_Graphics_Canvas_CanvasDevice_MaximumBitmapSizeInPixels.htm

**Comment by shawnhar** [comment]
This is fixed in Win2D 1.12.0 (released today)
